proposals/improved-definite-assignment.md

```diff
@@ -101,10 +101,11 @@ We introduce a new section **?. (null-conditional operator) expressions**. See t
 
 As in the definite assignment rules linked above, we refer to a given initially unassigned variable as *v*.
 
-We introduce the concept of "directly contains". An expression *E* is said to "directly contain" a subexpression *E<sub>1</sub>* if one of the following conditions holds:
+We introduce the concept of "directly contains". An expression *E* is said to "directly contain" a subexpression *E<sub>1</sub>* if it is not subject to a [user-defined conversion](../spec/conversions.md#user-defined-conversions) whose parameter is not of a non-nullable value type, and one of the following conditions holds:
 - *E* is *E<sub>1</sub>*. For example, `a?.b()` directly contains the expression `a?.b()`.
 - If *E* is a parenthesized expression `(E2)`, and *E<sub>2</sub>* directly contains *E<sub>1</sub>*.
 - If *E* is a null-forgiving operator expression `E2!`, and *E<sub>2</sub>* directly contains *E<sub>1</sub>*.
+- If *E* is a cast expression `(T)E2`, and the cast does not subject E<sub>2</sub> to a non-lifted user-defined conversion, and E<sub>2</sub> directly contains E<sub>1</sub>.
```

Comment:
>E<sub>2</sub> [](start = 69, length = 13)

Nit: missing asterisks on expressions on this line

Reply:
```suggestion
- If *E* is a cast expression `(T)E2`, and the cast does not subject *E<sub>2</sub>* to a non-lifted user-defined conversion, and *E<sub>2</sub>* directly contains *E<sub>1</sub>*.
```